ReactiveCocoaTests/UIKit/UITextFieldSpec.swift

```diff
@@ -44,8 +44,10 @@ class UITextFieldSpec: QuickSpec {
 				latestValue = text
 			}
 
-			textField.sendActions(for: .editingChanged)
-			expect(latestValue) == textField.text
+			for event in UIControlEvents.editingEvents {
+				textField.sendActions(for: event)
+				expect(latestValue) == textField.text
```

Comment:
Since this expectation is the same on every iteration, it doesn't seem like this change adds much.

We should probably improve the test here.